docs/web/docs/guides/how_to_use/efficiency_organization/reusing_configs.md

```diff
@@ -4,4 +4,61 @@ sidebar_position: 1
 
 # Use the same configs across tasks
 
-TODO - talk about setting up hydra profiles
+As you begin launching many Mephisto tasks, you may find that there are some specific argument options that you frequently use _across_ multiple tasks. Mephisto provides a way to reuse these types of configurations with `profile`s.
+
+Setting up profiles is pretty easy, and makes it so that you don't have to be writing architect and crowdprovider args on every launch:
+```
+python run_task.py mephisto/architect=heroku mephisto/provider=mturk_sandbox mephisto.provider.requester_name=MY_REQUESTER
+```
+
+Instead you can move these common configurations into a file in your `~/.mephisto/hydra_configs/profile` dir. 
```

Comment:
Maybe a provocative question, but is there any reason to have `hydra_configs` as part of the path? In the example folders we also have things like `parlai_chat_task_demo/hydra_configs/conf/`.... always as a nested folder. seems unnecessary to have the double besting. Perhaps `hydra_config` is part of the defaults but maybe we can use something straightforward here? this should also be easier now that we control the decorator.

Perhaps just `conf/`, e.g `~/.mephisto/conf/` that houses the profiles? or `~/.mephisto/profiles/`

Reply:
It's a hydra requirement, you need to point to a root folder, and then all sub folders of that folder are possible configuration options that can be set. As such it needs its own folder, or else hydra will consider all of the things present possible arguments. Agreed that it's not ergonomic, but I'm not sure how else we do it.